src/Scripting/CSharp/Hosting/ObjectFormatter/CSharpPrimitiveFormatter.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Scripting.Hosting;
+
+namespace Microsoft.CodeAnalysis.CSharp.Scripting.Hosting
+{
+    using static ObjectFormatterHelpers;
+
+    public class CSharpPrimitiveFormatter : CommonPrimitiveFormatter
+    {
+        protected override string NullLiteral => ObjectDisplay.NullLiteral;
+
+        protected override string FormatLiteral(bool value)
+        {
+            return ObjectDisplay.FormatLiteral(value);
+        }
+
+        protected override string FormatLiteral(string value, bool quote, bool useHexadecimalNumbers = false)
+        {
+            var options = ObjectDisplayOptions.None;
```

Comment:
It seems like maybe ```GetObjectDisplayOptions``` could be updated to handle this and the method below.  Specifically, I'm thinking something like:

```
ObjectDisplayOptions GetObjectDisplayOptions(bool useHex, bool quote = false, bool includeCodePoints = false)
```